### ISSUE TITLE
OverlayLayout doesn't work with non-specific SizeLayout

### ISSUE THREAD

**Issue by cooksimo** [linkedin/LayoutKit#170]
Using a SizeLayout with a constrained height, but no specified width with an OverlayLayout results in the internal overlays having a width of 0

```
let view = SizeLayout<UIView>(height: 165, config: { view in
           view.backgroundColor = .red
        })

        let overlayContent = InsetLayout(insets: UIEdgeInsets(top: 2.0, left: 5.0, bottom: 2.0, right: 5.0), alignment: .topLeading, sublayout: LabelLayout(text: "Test label", numberOfLines: 1, config: { label in
            label.textColor = .white
        }), config: nil)

        let overlay = OverlayLayout(primary: view, overlay: [overlayContent], config: nil)

        let topStack = StackLayout(axis: .vertical, spacing: 8.0, sublayouts: [openHomesOverlay], config: nil)
```

**Comment by Candyroot** [comment]
I think this bug has been resolved in pull request #155. I verified it using the latest version and cannot reproduce it.

**Comment by cooksimo** [comment]
I think you're right - will close this, thanks!
